extra-cpp-bindings/src/lib.rs

```diff
@@ -638,9 +646,34 @@ fn walletconnect_new_client(
 unsafe impl Send for ffi::WalletConnectCallback {}
 unsafe impl Sync for ffi::WalletConnectCallback {}
 
+fn generate_qrcode(qrcodestring: String) -> Result<crate::ffi::WalletQrcode> {
+    let result: Vec<Vec<bool>> = qrcode_generator::to_matrix(qrcodestring.clone(), QrCodeEcc::Low)?;
```

Comment:
Why to matrix and then iterating over it instead of encoding to a raw image directly: https://docs.rs/qrcode-generator/latest/qrcode_generator/fn.to_image.html ?

Reply:
will change to use image